package.json

```diff
@@ -36,8 +36,8 @@
     "db:start": "./start-couchdb",
     "deploy": "./deploy",
     "pretest": "standard && npm run db:start && sleep 5",
-    "start": "node --harmony_async_await index.js",
```

Comment:
This goes with the engine-changes, should be separate from the main commit here.